vocode/streaming/agent/state_agent.py

```diff
@@ -162,38 +105,11 @@ def translate_to_english(
 async def handle_question(
     state,
     go_to_state: Callable[[str], Awaitable[Any]],
-    speak: Callable[[str], None],
-    get_question_text: Callable[[None], Awaitable[str]],
-    call_ai: Callable[[str, str, str], Awaitable[str]],
     logger: logging.Logger,
 ):
-    question_text = await get_question_text()
-
     async def resume(human_input):
         logger.info(f"continuing at {state['id']} with user response {human_input}")
-        branch_decision, ai_response = await on_user_input_within_state(
-            state, human_input, question_text, call_ai, logger
-        )
-
-        if branch_decision == BranchDecision.PAUSE:
-            if ai_response:
-                speak(ai_response)
-
-            async def get_stored_question_text():
-                return question_text
-
-            return await handle_question(
-                state=state,
-                go_to_state=go_to_state,
-                speak=speak,
-                call_ai=call_ai,
-                logger=logger,
-                get_question_text=get_stored_question_text,
-            )
-        if branch_decision == BranchDecision.CONTINUE:
-            return await go_to_state(get_default_next_state(state))
-        if branch_decision == BranchDecision.SWITCH:
-            return await go_to_state("start")
+        return await go_to_state(get_default_next_state(state))
```

Comment:
guessing this is the line that requires us to inject an `options` after each question